source/fem-dem/vans_assemblers.cc

```diff
@@ -1767,9 +1771,9 @@ VANSAssemblerVorticalTorque<dim>::calculate_particle_fluid_interactions(
     RequiresConstantDensity(
       "VANSAssemblerVorticalTorque<dim>::calculate_particle_fluid_interactions"));
 
-  const double density = scratch_data.properties_manager.get_density_scale();
-  const double kinematic_viscosity =
-    scratch_data.properties_manager.get_kinematic_viscosity_scale();
+  const std::vector<double> density = scratch_data.density_at_particle_location;
+  const std::vector<double> kinematic_viscosity =
```

Comment:
```suggestion
  const std::vector<double>& density = scratch_data.density_at_particle_location;
  const std::vector<double>& kinematic_viscosity =
```
Both should be references to avoid copying.